src/data/repositories/CountryInformationDefaultRepository.ts

```diff
@@ -42,8 +43,8 @@ export class CountryInformationDefaultRepository implements CountryInformationRe
                 const regionName =
                     orgUnits.find(ou => ou.id !== countryId && ou.level === countryLevel - 1)?.shortName || "";
 
-                const enrollment = tei?.enrollments[0];
-                const events = tei?.enrollments[0]?.events || [];
+                const enrollment = tei?.enrollments?.[0];
+                const events = tei?.enrollments?.[0]?.events || [];
```

Comment:
This can be `const events = enrollment?.events || [];` also

Reply:
Sorry I didn't understand this one

Reply:
ah this is something minor, it's just because `const enrollment = tei?.enrollments?.[0];` you don't need to repeat in events the same part `const events = tei?.enrollments?.[0]?.events || [];` so you can replace it by `const events = enrollment?.events || [];`. But this has zero importance

Reply:
Oh ok makes sense, I've done that change now